resources/Makefile.erb

```diff
@@ -1,11 +1,18 @@
 <% dirnames = get_directories.map {|d| d.path } %>
-export SHELL := $(shell which bash)
 
-tempdir := $(shell mktemp -d -p /var/tmp 2>/dev/null || mktemp -d -t 'tmp')
 workdir := $(PWD)
 
+export SHELL := "${workdir}/profiling_shell.sh"
```

Comment:
we shouldn't use `export` here if we don't actually have to, and the value after the assignment doesn't need to be quoted. 

```make
# This should be more than sufficient to set a valid shell for Make to send commands to
SHELL = $(PWD)/profiling_shell.sh
```